contentcuration/contentcuration/frontend/channelEdit/views/trash/TrashModal.vue

```diff
@@ -190,14 +180,19 @@
         return sortBy(this.getContentNodeChildren(this.trashId), 'modified').reverse();
       },
       backLink() {
-        const lastIndex = Math.max(0, this.$route.matched.length - 2);
         return {
-          name: this.$route.matched[lastIndex].name,
-          query: this.$route.query,
+          name: RouterNames.TREE_VIEW,
```

Comment:
I am not sure what was the motivation behind

```
const lastIndex = Math.max(0, this.$route.matched.length - 2);
```

It seems that the trash modal can be currently visited only from the root tree view, or are there any other places from where it can be accessed?